### PR TITLE
Added the LeaderBoard setUp markUp, styles and Javascript

### PR DESCRIPTION
**Pull request by Asiimwe-Mark**
I added the leaderBoard markup styles and javaScript. I used gitflow, ES6 modules and webpack to do this project following the project requirements.

Project requirements
IMPORTANT NOTE This is the first time you will use [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). Keep that in mind!
Create an npm project with webpack. (If you need a refresher, check the previous project or webpack exercise).
Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps. Use this wireframe as a guide:
Don't make API calls (you'll do it in a following lesson).
Use ES6 modules, with import and export.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,102 @@
...
+
+### Tech Stack **HTML, CSS and JavaScript**
+### Key Features <a name="key-features"></a>
+- **Contains a LeaderBoard project that takes and stores game scores for players items as ojects in an array**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### LiveDemo
+[Live Demo] You can see the live Demo of the website here  
+
```
Comment (Efiamotu-1) [inline README.md]: - [ ] Your project currently does not have a live demo, you are either required to add the correct link to the live demo or remove this section. 👍 
- Make sure that if you are removing this section you also remove from the header links above in your readme file 👍 ✨

File: README.md [hunk 2]
```diff
@@ -1 +1,102 @@
...
+### Install
+
+Install this project with: - npm install to install local dependencies - npm run build to build it
+
+### Usage
+
+To run the project : npm start
+
+### Project_presentation
+[Project Presentation] You can see the live presentation of the key concepts taken into account while creating this project here:
```
Comment (Efiamotu-1) [inline README.md]: - [ ] Your project currently does not have a project presentation, you are either required to add the correct link to the project presentation or remove this section. 👍 
- Make sure that if you are removing this section you also remove from the header links above in your readme file 👍 ✨

File: src/index.html [hunk 3]
```diff
@@ -3,9 +3,29 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <title>Leaderboard</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <main>
+        <h1>Leaderboard</h1>
```
Comment (Efiamotu-1) [inline src/index.html:10]: - [ ] As shown in the image below, your project design is missing the section highlighted in red, you are required to add this section as it is part of the wireframe design given. It is important to pay attention to details when designing as a frontend developer. 👍 ✨ 
- project wireframe
wf

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @As1imwe-Mark, 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ Good job setting up webpack
- ✔️ Good UI design
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by souad988 — APPROVED** [review]
Hi @As1imwe-Mark , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved2


## Highlights

Use of webPack to bundel the project
Good UI implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**